mule-extensions-api/src/main/java/org/mule/runtime/extension/api/introspection/property/DisplayModelProperty.java

```diff
@@ -24,22 +24,46 @@
     private final int order;
     private final String groupName;
     private final String tabName;
+    private final String summary;
```

Comment:
I think that summary should be a model property of its own. We might want to add summary to operations , where things like text and tabName makes no sense